skbio/io/format/tests/test_embl.py

```diff
@@ -0,0 +1,858 @@
+            obs = fh.getvalue()
+
+        with open(self.multi_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+    # TODO: add support for protein
+    # current status of protein support is described in issue-1499
+    # https://github.com/biocore/scikit-bio/issues/1499
+    def test_protein_to_embl(self):
+        seq, md, imd, constructor = self.protein
+        obj = constructor(seq, md, interval_metadata=imd)
+
+        with io.StringIO() as fh:
+            self.assertRaisesRegex(EMBLFormatError,
+                                   "There's no protein support for EMBL "
+                                   "record",
+                                   _protein_to_embl, [obj], fh)
+
+    def test_rna_to_embl(self):
+        with io.StringIO() as fh:
+            seq, md, imd, constructor = self.single_rna
+            obj = constructor(seq, md, interval_metadata=imd, lowercase=True)
+            _rna_to_embl(obj, fh)
+            obs = fh.getvalue()
+
+        with open(self.single_rna_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+    def test_rna_to_embl_flp(self):
+        """Test writing feature level products"""
+
+        with io.StringIO() as fh:
+            seq, md, imd, constructor = self.feature_level
+            obj = constructor(seq, md, interval_metadata=imd, lowercase=True)
+            _rna_to_embl(obj, fh)
+            obs = fh.getvalue()
+
+        with open(self.feature_level_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+
+class RoundtripTests(EMBLIOTests):
+    def test_roundtrip_generator(self):
+        with io.StringIO() as fh:
+            _generator_to_embl(_embl_to_generator(self.multi_fp), fh)
+            obs = fh.getvalue()
+
+        with open(self.multi_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+    def test_roundtrip_rna(self):
+        with io.StringIO() as fh:
+            _rna_to_embl(_embl_to_rna(self.single_rna_fp), fh)
+            obs = fh.getvalue()
+
+        with open(self.single_rna_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+    def test_roundtrip_dna(self):
+        with io.StringIO() as fh:
+            _dna_to_embl(_embl_to_dna(self.single_rna_fp), fh)
+            obs = fh.getvalue()
+
+        with open(self.single_rna_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+    # TODO: test_roundtrip_protein
+    # current status of protein support is described in issue-1499
+    # https://github.com/biocore/scikit-bio/issues/1499
+
+    def test_roundtrip_sequence(self):
+        with io.StringIO() as fh:
+            _sequence_to_embl(_embl_to_sequence(self.single_rna_fp), fh)
+            obs = fh.getvalue()
+
+        with open(self.single_rna_fp) as fh:
+            exp = fh.read()
+
+        self.assertEqual(obs, exp)
+
+
+class Convertertest(EMBLIOTests):
```

Comment:
Is not so simple. When I write a `genbank` record, I put a locus like this `LOCUS       M14399   63 bp   mRNA   linear   PRO   02-SEP-1999`. However the embl `PRO` division is not contemplated by [genbank parser](https://github.com/bunop/scikit-bio/blob/issue-1483/skbio/io/format/genbank.py#L565). I need to translate division in genbank division (not sure if it will be always possible). Could I modify `genbank` to read any three letter code?